napari/_qt/qt_event_loop.py

```diff
@@ -142,6 +153,11 @@ def get_app(
             show_console_notification
         )
 
+    app.setWindowIcon(QIcon(kwargs.get('icon')))
```

Comment:
This may clobber user specified window icons

Reply:
yeah :/ ... can revert.  The issue I was trying to address was that if someone closes the viewer with cmd-Q ... then the icon will be gone forever unless I do this in `get_app`.  I can try to check first to make sure there is an icon before clobbering.

Reply:
fixed now.  thanks!